src/components/molecules/ChatsList/ChatsList.scss

```diff
@@ -34,15 +34,40 @@ $primary: #005ee5;
         width: 100%;
```

Comment:
note: I'm not deeply code reviewing the styles here, i'm mostly assuming that if they work in testing/clickthrough they should be fine.